x/wasm/binding/cw_struct/cw_msg.go

```diff
@@ -0,0 +1,53 @@
+package cw_struct
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+type BindingMsg struct {
+	OpenPosition          *OpenPosition          `json:"open_position,omitempty"`
```

Comment:
Is this crucial to be pointer?

Reply:
https://github.com/NibiruChain/nibiru/pull/1287#discussion_r1176968065